deploy/crds/devconsole_v1alpha1_gitsourceanalysis_crd.yaml

```diff
@@ -9,6 +9,8 @@ spec:
     listKind: GitSourceAnalysisList
     plural: gitsourceanalyses
     singular: gitsourceanalysis
+    shortNames:
+      - srcan
```

Comment:
If the other shortname is "gs" maybe this can be "gsa" or "gsan" or something?

Reply:
"gs" and "gsa" sounds good.

Reply:
👍 
done with 29fe71b